Makefile

```
@@ -96,9 +96,7 @@ dist-test: dist-zip install-vendor
 # install vendor rules
 ###
 install-vendor:
-	if [[ ! -f vendor/autoload.php && ! -f ../../../vendor/autoload.php ]]; then \
-		$(COMPOSER) install $(COMPOSER_INSTALL_FLAGS) $(VENDOR_FLAGS)
-	fi
+	$(COMPOSER) install $(COMPOSER_INSTALL_FLAGS) $(VENDOR_FLAGS)
```

Actually, we could let make do the magic
```suggestion
install-vendor: vendor/autoload.php
vendor/autoload.php: composer.json
	$(COMPOSER) install $(COMPOSER_INSTALL_FLAGS) $(VENDOR_FLAGS)
```